docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -500,28 +500,28 @@ initially in tree[I].Len; the codes are produced in tree[I].Code.
        code length:
 
 .KS
-          code = 0;
-          bl_count[0] = 0;
-          for (bits = 1; bits <= MAX_BITS; bits++) {
-             code = (code + bl_count[bits-1]) << 1;
-             next_code[bits] = code;
-          }
+         code = 0;
```

Comment:
I could be wrong, but I count 9-spaces in my patch and 10-spaces in the original. The reason I removed the space was because I did notice that the spec used 3-space tabs consistently (for the most part).

https://github.com/google/brotli/blob/master/docs/draft-alakuijala-brotli-07.txt#L555-L574
You can see that the "2)" does not line up with the "3)".